invenio_communities/utils.py

```diff
@@ -64,6 +70,15 @@ def on_membership_change(identity=None):
         current_cache.delete(identity_cache_key(identity))
 
 
+def delete_all_user_communities_keys():
+    """Handler called when a group is updated/added/removed from a community."""
+    binary_keys = current_cache.cache._read_client.keys(f"*{IDENTITY_KEY}*")
```

Comment:
This is not ideal, but there is no way to fetch concrete keys from flask-caching. That's why we are accessing protected methods.